src/cargo/ops/cargo_package.rs

```diff
@@ -331,6 +332,29 @@ fn build_ar_list(
             warn_on_nonexistent_file(&pkg, &readme_path, "readme", &ws)?;
         }
     }
+
+    for t in pkg
+        .manifest()
+        .targets()
+        .iter()
+        .filter(|t| t.is_custom_build())
```

Comment:
should i handle other target type and how can i test on them?

Reply:
I would say yes,, as not every `target.path` are required to be packaged. We don't want to hassle everybody with their excluded examples.

But maybe leave that for follow-ups. We can always iterate on it later on :)

Reply:
Thanks, this time i will focus on the `custom build target`.